gitignore.js

```diff
@@ -6,12 +6,15 @@ import gitIgnore from 'ignore';
 import slash from 'slash';
 import {toPath} from './utilities.js';
 
-const DEFAULT_IGNORE = [
-	'**/node_modules/**',
-	'**/flow-typed/**',
-	'**/coverage/**',
-	'**/.git',
-];
+const gitignoreGlobOptions = {
+	ignore: [
+		'**/node_modules',
+		'**/flow-typed',
+		'**/coverage',
+		'**/.git',
```

Comment:
As I tested, when use `**/node_modules/` and `**/node_modules/**/*`, `fast-glob` will read those directories, but when use `**/node_modules` , `fast-glob` won't read them.

Reply:
Answering this question https://github.com/sindresorhus/globby/pull/224#discussion_r788528852